wslpy/exec.py

```diff
@@ -3,6 +3,42 @@
 This is the execution class to execute commands from different windows executables
 """
 import subprocess
+import os
+from wslpy.__init__ import isWSL
+import sys
+
+
+def __isInteropEnabled__():
+    cat_wslInterop = subprocess.getoutput(
+        "cat /proc/sys/fs/binfmt_misc/WSLInterop | grep '.*abled'")
+    return cat_wslInterop
+
+
+def isInteropEnabled():
+    """
+    Checks if interoperablility is enabled.
+
+    Returns
+    _______
+    A boolean value, `True` if interoperablility is enabled.
+    """
+    if __isInteropEnabled__() == 'enabled':
+        return True
+    else:
+        return False
+
+
+def preCheck():
+
+    if isWSL():
+        if isInteropEnabled():
+            pass
+        else:
+            sys.exit(
+                "Please enable interop /n Use 'echo 1 > /proc/sys/fs/binfmt_misc/WSLInterop'")

```

Comment:
You can just do `if not isInteropEnabled()`

Reply:
Fogive me sire for my rookie mistakes